x-pack/test/functional/apps/ml/data_visualizer/index_data_visualizer_grid_in_dashboard.ts

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { TestData, MetricFieldVisConfig } from './types';
+import { farequoteLuceneFiltersSearchTestData } from './index_test_data';
+
+const SHOW_FIELD_STATISTICS = 'discover:showFieldStatistics';
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const PageObjects = getPageObjects(['common', 'discover', 'timePicker', 'settings', 'dashboard']);
+  const ml = getService('ml');
+  const testSubjects = getService('testSubjects');
+  const retry = getService('retry');
+  const toasts = getService('toasts');
+  const dashboardAddPanel = getService('dashboardAddPanel');
+
+  const selectIndexPattern = async (indexPattern: string) => {
+    await retry.tryForTime(2 * 1000, async () => {
+      await PageObjects.discover.selectIndexPattern(indexPattern);
+      const indexPatternTitle = await testSubjects.getVisibleText('indexPattern-switch-link');
+      expect(indexPatternTitle).to.be(indexPattern);
+    });
+  };
+
+  const clearAdvancedSetting = async (propertyName: string) => {
+    await retry.tryForTime(2 * 1000, async () => {
+      await PageObjects.common.navigateToUrl('management', 'kibana/settings', {
+        shouldUseHashForSubUrl: false,
+      });
+      if ((await PageObjects.settings.getAdvancedSettingCheckbox(propertyName)) === 'true') {
+        await PageObjects.settings.clearAdvancedSettings(propertyName);
+      }
+    });
+  };
+
+  const setAdvancedSettingCheckbox = async (propertyName: string, checkedState: boolean) => {
+    await retry.tryForTime(2 * 1000, async () => {
+      await PageObjects.common.navigateToUrl('management', 'kibana/settings', {
+        shouldUseHashForSubUrl: false,
+      });
+      await testSubjects.click('settings');
+      await toasts.dismissAllToasts();
+      await PageObjects.settings.toggleAdvancedSettingCheckbox(propertyName, checkedState);
+    });
+  };
+
+  function runTests(testData: TestData) {
+    const savedSearchTitle = `Field stats for ${testData.suiteTitle} ${Date.now()}`;
+    const dashboardTitle = `Dashboard for ${testData.suiteTitle} ${Date.now()}`;
+
+    describe(`with ${testData.suiteTitle}`, function () {
+      after(async function () {
+        await ml.testResources.deleteSavedSearchByTitle(dashboardTitle);
```

Comment:
Should delete `savedSearchTitle` here.

Reply:
Fix here https://github.com/elastic/kibana/pull/116774/commits/f8b6acf943892d049d7951bedb4aa6ddad25aff6